src/helpers/setToolModeAndGroup.js

```diff
@@ -5,6 +5,15 @@ import selectors from 'selectors';
 
 export default (store, toolName) => {
   const { dispatch, getState } = store;
+
+  const openElements = selectors.getOpenElements(getState());
```

Comment:
I think a better place to put this logic is at 
https://github.com/PDFTron/webviewer-ui/blob/9d85f21e0f9efe8febf74d6b71f06886ef200a62/src/helpers/hotkeysManager.js#L408-L419
I think having code in this above function is more intuitive to think. We want to disable tool hotkeys when the modal is open, not still allow hotkeys but make an exception for the modal in this function. What do you think?

Reply:
I moved the code to the `createToolHotkeyHandler` function in `hotkeysManager`